pyproject.toml

```diff
@@ -125,4 +126,4 @@ target-version = ["py38", "py39", "py310", "py311", "py312"]
 
 [build-system]
 requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+build-backend = "poetry.core.masonry.api"
```

Comment:
No new line at end of file

Reply:
Yes, but the same style pattern used in the main branch or any other branches: https://github.com/MobileTeleSystems/RecTools/blob/main/pyproject.toml#L128

So, is it really necessary in this case?

Reply:
I see you point. It' kinda strange it shows on the diff in this particular PR